src/wasm-lib/kcl/src/std/loft.rs

```diff
@@ -116,28 +105,24 @@ pub async fn loft(exec_state: &mut ExecState, args: Args) -> Result<KclValue, Kc
 /// const circleSketch1 = startSketchOn(offsetPlane('XY', 150))
 ///     |> circle({ center = [0, 100], radius = 20 }, %)
 ///
-/// loft([squareSketch, circleSketch0, circleSketch1], {
-///     // This can be set to override the automatically determined
-///     // topological base curve, which is usually the first section encountered.
-///     baseCurveIndex = 0,
-///     // Attempt to approximate rational curves (such as arcs) using a bezier.
-///     // This will remove banding around interpolations between arcs and non-arcs.
-///     // It may produce errors in other scenarios Over time, this field won't be necessary.
-///     bezApproximateRational = false,
-///     // Tolerance for the loft operation.
-///     tolerance = 0.000001,
-///     // Degree of the interpolation. Must be greater than zero.
-///     // For example, use 2 for quadratic, or 3 for cubic interpolation in
-///     // the V direction. This defaults to 2, if not specified.
-///     vDegree = 2,
-/// })
+/// loft([squareSketch, circleSketch0, circleSketch1],
+///     baseCurveIndex: 0,
```

Comment:
Had to remove comments due to https://github.com/KittyCAD/modeling-app/issues/4763